torch/quantization/_numeric_suite.py

```diff
@@ -93,6 +93,22 @@ def compare_weights(float_dict, quantized_dict):
             weight_dict[key]["float"] = float_dict[match_key]
             weight_dict[key]["quantized"] = quantized_dict[key][0]
 
+        # For LSTM
+        split_str = key.split(".")
+        if split_str[-1] == "param" and split_str[-3] == "_all_weight_values":
+            layer = split_str[-2]
+            module_name = ".".join(split_str[:-3])
+            float_weight_ih_key = module_name + ".weight_ih_l" + layer
+            float_weight_hh_key = module_name + ".weight_hh_l" + layer
+            print('lstm float ih key is: ', float_weight_ih_key)
+            print('lstm float hh key is: ', float_weight_hh_key)
+            if float_weight_ih_key in float_dict and float_weight_hh_key in float_dict:
+                weight_dict[key] = {}
+                weight_dict[key]["float"] = float_dict[float_weight_ih_key]
```

Comment:
This feels hacky, I am going to write a PR to make it easy to get weights for LSTMs

Reply:
Can we land this PR first to have a working prototype? Once you made the changes I can update it to use the new way to get weights.